packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/address/move-eligibility-address.html

```diff
@@ -0,0 +1,270 @@
+<form name="addressCollectForm" data-ng-submit="$ctrl.submitAddress()">
+    <div class="form-group">
+        <label for="zipCodeElem" data-translate="pack_move_eligibility_zipcode">
+        </label>
+        <input
+            type="text"
+            class="form-control"
+            id="zipCodeElem"
+            name="zipCode"
+            data-translate-attr="{ placeholder: 'pack_move_eligibility_zipcode' }"
+            data-ng-model="$ctrl.address.zipCode"
+            data-ui-validate="'$ctrl.validator.tucIsZipcode($value, [\'metropolitanFrance\'])'"
+            data-ng-change="$ctrl.getCities($ctrl.address.zipCode)"
+            data-ng-disabled="$ctrl.loaders.cities || $ctrl.method !== 'address'"
+            required
+        />
+    </div>
```

Comment:
**suggestion**: `oui-field` can be used here. (See: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-field-webcomponent--default)